fix(2021.1): update links to javadoc (#1448)
bonitasoft/bonita-doc: modules/ROOT/pages/index.adoc

== Getting started
[.card.card-index]
--

> xref:what-is-bonita.adoc[[.card-title]#Getting started tutorial# [.card-body.card-content-overflow]#pass:q[A step-by-step creation of a first simple Living Application to learn about all the concepts in Bonita.]#]

Reviewer: Why not keep getting-started-tutorial.adoc ?

Author: This page does not exist and didn't exist in the markdown format. The original link was `[Getting started tutorial](_getting-started-tutorial.md)` (see the latest commit in markdown: https://github.com/bonitasoft/bonita-doc/blob/d48c88796264584f02a65e00b56d140e178859bd/md/index.md)
All pages starting with `_` were generated from the taxonomy and the generated page was a list of links, so not very engaging.
Here, I have put the link to the 1st page of the tutorial, also available via the taxonomy.

Author: hm, after checking the preview, the `tutorial-overview` page seems more accurate for this link
Notes:
- `what-is-bonita` is the 1st page in the taxonomy in Bonita doc 7.9 and 7.10
- 7.11 and later versions have the `tutorial-overview` page as getting started entry point

Author: The getting started link will be managed by #1456, I am going to revert the change in this PR

> xref:what-is-bonita.adoc[[.card-title]#Getting started tutorial# [.card-body.card-content-overflow]#pass:q[A step-by-step creation of a first simple Living Application to learn about all the concepts in Bonita.]#]

Author: ```suggestion
xref:tutorial-overview.adoc[[.card-title]#Getting started tutorial# [.card-body.card-content-overflow]#pass:q[A step-by-step creation of a first simple Living Application to learn about all the concepts in Bonita.]#]
```
